src/main/java/com/notably/commons/core/path/AbsolutePath.java

```diff
@@ -1,53 +1,113 @@
+            if (absolutePathString.contains("../")) {
+                return false;
+            } else {
+                return true;
+            }
+        }
         return false;
     }
 
     /**
-     * TODO: Add Javadoc
+     * Creates an absolute path from a string.
+     * @param absolutePathString used to create an absolutePath.
+     * @return Absolute Path.
+     * @throws InvalidPathException if String provided is not a valid absolutePath.
      */
     public static AbsolutePath fromString(String absolutePathString) throws InvalidPathException {
-        // TODO: Add implementation
-        return new AbsolutePath();
+        if (isValidAbsolutePath(absolutePathString)) {
+            return new AbsolutePath(absolutePathString);
+        } else {
+            throw new InvalidPathException(INVALIDABSOLUTEPATH);
+        }
     }
 
     /**
-     * TODO: Add Javadoc
+     * Convert from relative path to absolute path.
+     * @param relativePath used to convert to absolute path.
+     * @param currentWorkingPath of the current working directory.
+     * @return the converted AbsolutePath.
+     * @throws InvalidPathException
      */
     public static AbsolutePath fromRelativePath(RelativePath relativePath,
             AbsolutePath currentWorkingPath) throws InvalidPathException {
-        // TODO: Add implementation
-        return new AbsolutePath();
+        List<String> temp = new ArrayList<>(relativePath.getComponents());
+        List<String> temp2 = new ArrayList<>(currentWorkingPath.getComponents());
+        temp2.addAll((temp));
+        return new AbsolutePath(String.join("/", temp2));
     }
 
     /**
-     * TODO: Add Javadoc
+     * Convert Absolute path to Relative path.
+     * @param currentWorkingPath of the current working directory.
+     * @return the converted RelativePath
      */
     public RelativePath toRelativePath(AbsolutePath currentWorkingPath) {
         // TODO: Add implementation
-        return new RelativePath();
+        int sizeOfCurrent = currentWorkingPath.getComponents().size();
+        List<String> temp = this.paths.subList(sizeOfCurrent, this.paths.size());
+        String relativePathString = String.join("/", temp);
+        return new RelativePath(relativePathString);
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
     public List<String> getComponents() {
-        // TODO: Add implementation
-        return List.of();
+        return this.paths;
+    }
+
+    @Override
+    public boolean equals(Object object) {
+        if (!(object instanceof Path)) {
+            return false;
+        }
+
+        Path another = (Path) object;
+        List<String> temp = another.getComponents();
+        return this.paths.equals(temp);
+    }
+    @Override
+    public int hashCode() {
+        return Objects.hash(this.paths);
+    }
+
+    @Override
+    public String toString() {
+        return (Arrays.toString(this.paths.toArray()));
```

Comment:
I don't think you need to do this. By default, `List`'s `toString()` method is already propertly formatted. That is, even if you only return `paths.toString` here, everything will just look as good.

Nevertheless, perhaps you would want to consider joining the `paths` list with a `"/"` character and return the result string instead?